cmd/thanos/sidecar.go

```diff
@@ -305,6 +322,13 @@ func (s *extLabelSet) GetPB() []storepb.Label {
 	return lset
 }
 
+func (s *metadata) GetTimestamps() (mint int64, maxt int64) {
+	s.mtx.Lock()
+	defer s.mtx.Unlock()
```

Comment:
why `defer` here, but not in the similar code ~30 lines earlier?

Reply:
fixed